app/core/ViewLoad.coffee

```diff
@@ -0,0 +1,65 @@
+VIEW_LOAD_LOG = false
+SHOW_NOTY = false
+
+class ViewLoad
```

Comment:
ViewLoad isn't super self-descriptive name; maybe something with "tracker" or "timer" etc in the name. "ViewLoadTimer" feels better.